modules/3d/include/opencv2/3d.hpp

```diff
@@ -2800,10 +2833,35 @@ CV_EXPORTS_W void loadMesh(const String &filename, OutputArray vertices, OutputA
 * @param filename Name of the file.
 * @param vertices (vector of Point3f) vertex coordinates of a mesh
 * @param normals (vector of Point3f) vertex normals of a mesh
+* @param colors (vector of Point3f) vertex colors of a mesh
+* @param indices (vector of vectors of int) vertex normals of a mesh
+*/
+CV_EXPORTS_W void saveMesh(const String &filename, InputArray vertices, InputArray normals, InputArray colors, InputArrayOfArrays indices);
+
+/** @overload
+*  @brief Saves a mesh to a specified file.
+*
+* The function saves mesh to the specified file.
+* File format is chosen based on the filename extension.
+*
+* @param filename Name of the file.
+* @param vertices (vector of Point3f) vertex coordinates of a mesh
+* @param normals (vector of Point3f) vertex normals of a mesh
 * @param indices (vector of vectors of int) vertex normals of a mesh
 */
 CV_EXPORTS_W void saveMesh(const String &filename, InputArray vertices, InputArray normals, InputArrayOfArrays indices);
 
+/** @overload
+*  @brief Saves a mesh to a specified file.
+*
+* The function saves mesh to the specified file.
+* File format is chosen based on the filename extension.
+*
+* @param filename Name of the file.
+* @param vertices (vector of Point3f) vertex coordinates of a mesh
+* @param indices (vector of vectors of int) vertex normals of a mesh
+*/
+CV_EXPORTS_W void saveMesh(const String &filename, InputArray vertices, InputArrayOfArrays indices);
```

Comment:
Why do we need overloads. Looks like the most complete function with default values is enough, e.g.:
```
CV_EXPORTS_W void saveMesh(const String &filename, InputArray vertices,  InputArrayOfArrays indices, InputArray normals = noArray(), colors = noArray());
```

Reply:
Didn't want to change the existing API;
But you're right, this one looks better, fixing it